Refresh AITK Playground Page

MuyangAmigo opened the pull request:
- Refresh playground page
- Add instructions for deploying models directly from playground to AI Foundry
- Add missing contents (chat input, model responses, comparison etc)

ntrogh commented on docs/intelligentapps/playground.md:
```diff
@@ -1,40 +1,120 @@
+
+### Configure inference parameters
+The available parameters depend on the model type and publisher. Common parameters include:
+- **Temperature**: Controls the randomness of the model's output. Lower values make the output more deterministic. Higher values introduce more variability.
```
No need to make these bold.

MuyangAmigo replied:
Removed the bold style

ntrogh commented on docs/intelligentapps/playground.md:
```diff
@@ -1,40 +1,120 @@
+
+![Chat prompt input box in AI Toolkit playground showing an attachment icon for adding files.](./images/playground/chat_prompt.png)
+
+The chat prompt is the input text you provide to the model. You can enter a question, statement, or any text that you want the model to respond to. The model will generate a response based on the provided prompt and the configured system prompts and parameters.
```
```suggestion
The chat prompt is the input text you provide to the model. You can enter a question, statement, or any text that you want the model to respond to. The model generates a response based on the provided prompt and the configured system prompts and parameters.
```

ntrogh commented on docs/intelligentapps/playground.md:
```diff
@@ -1,40 +1,120 @@
+## Model responses
+![AI Toolkit playground response area showing a generated Python code snippet for encoding text in base64, with options to copy or regenerate the response.](./images/playground/response_area.png)
+
+Model responses are displayed above the chat prompt input box. Before you enter a chat prompt, the playground shows a welcome screen with three starter prompts for you to easily get started with the playground.
```
```suggestion
Model responses are displayed above the chat prompt input box. Before you enter a chat prompt, the playground shows a welcome screen with three starter prompts for you to get started with the playground.
```

ntrogh commented on docs/intelligentapps/playground.md:
```diff
@@ -1,40 +1,120 @@
+Available features in the model response area:
+- **Copy Response**: Copy the model's response text to the clipboard.
+- **Regenerate Response**: Regenerate the model's response based on the same prompt or regenerate response with another model.
+- **Copy Code**: Copy the generated code in the response to the clipboard or inset it into a new file.
```
```suggestion
- **Copy Code**: Copy the generated code in the response to the clipboard or insert it into a new file.
```

ntrogh commented on docs/intelligentapps/playground.md:
```diff
@@ -1,40 +1,120 @@
 
-    ![Select a model and configure context instructions in the playground.](./images/playground/parameters.png)
+- **New Playground**: Create a new playground session.
+- **History**: View, rename, delete and switch to a chat history.
```
```suggestion
- **History**: View, rename, delete, and switch to a chat history.
```

ntrogh commented on docs/intelligentapps/playground.md:
```diff
@@ -1,40 +1,120 @@
-    ![Select a model and configure context instructions in the playground.](./images/playground/parameters.png)
+- **New Playground**: Create a new playground session.
+- **History**: View, rename, delete and switch to a chat history.
+- **Automatic Naming**: Naming of chat sessions based on the user prompt will be automatically generated.
```
```suggestion
- **Automatic Naming**: Names of chat sessions are automatically generated based on the user prompt.
```

ntrogh commented on docs/intelligentapps/playground.md:
```diff
@@ -1,40 +1,120 @@
+To compare model responses:
+1. Select the **Compare** button in the tool bar.
+2. Choose the models you want to compare from the dropdown list.
+3. Enter the same chat prompt in one of the input boxes. Your prompt will be copied to the other input boxes.
```
```suggestion
3. Enter the same chat prompt in one of the input boxes. Your prompt is copied to the other input boxes.
```

ntrogh commented on docs/intelligentapps/playground.md:
```diff
@@ -1,40 +1,120 @@
+1. Select the **Deploy to Azure AI Foundry** button in the warning message.
+2. In the dialog that appears, enter the required information, such as deployment name, deployment type, and deployment details.
+3. Select **Deploy** to start the deployment process.
+4. A pop-up appears to confirm the deployment and charges will be applied to your Azure subscription. Select **Deploy** to confirm.
```
```suggestion
4. A pop-up confirms the deployment and charges are applied to your Azure subscription. Select **Deploy** to confirm.
```

ntrogh reviewed (commented):
@MuyangAmigo Left some suggestions. Additional suggestion for the screenshots, zoom in one level in VS Code, because the text in the screenshots is tiny.

MuyangAmigo commented:
> @MuyangAmigo Left some suggestions. Additional suggestion for the screenshots, zoom in one level in VS Code, because the text in the screenshots is tiny.

@ntrogh Thank you for the review, I have addressed your comments. With your approval this page can also be merged.
